KwfBundle/EventListener/ExceptionHandler.php

```diff
@@ -44,6 +45,10 @@ public function onKernelException(GetResponseForExceptionEvent $event)
 
             $flattenException = FlattenException::create($exception);
             $event->setResponse($this->controller->showAction($request, $flattenException));
+        } else if ($exception instanceof AccessDeniedException) {
+            $event->stopPropagation();
+            \Kwf_Debug::handleException(new \Kwf_Exception_AccessDenied());
+            exit;
         } else if ($exception instanceof \Exception) {
             $event->stopPropagation();
             \Kwf_Debug::handleException($event->getException());
```

Comment:
in dem fall fände ich das sauberer

\Kwf_Debug::handleException($exception instanceof AccessDeniedException ? new \Kwf_Exception_AccessDenied() : $event->getException());